tools/harness-thci/OpenThread.py

```diff
@@ -1190,35 +1253,36 @@ def setPANID(self, xPAN):
             True: successful to set the Thread Network PAN ID
             False: fail to set the Thread Network PAN ID
         """
-        print '%s call setPANID' % self.port
-        print xPAN
+        print('%s call setPANID' % self.port)
+        print(xPAN)
         panid = ''
         try:
             if not isinstance(xPAN, str):
                 panid = str(hex(xPAN))
-                print panid
+                print(panid)
 
             cmd = 'panid %s' % panid
             datasetCmd = 'dataset panid %s' % panid
             self.hasActiveDatasetToCommit = True
-            return self.__sendCommand(cmd)[0] == 'Done' and self.__sendCommand(datasetCmd)[0] == 'Done'
-        except Exception, e:
-            ModuleHelper.WriteIntoDebugLogger("setPANID() Error: " + str(e))
+            return self.__sendCommand(cmd)[-1] == 'Done' and self.__sendCommand(
+                datasetCmd)[0] == 'Done'
+        except Exception as e:
+            ModuleHelper.WriteIntoDebugLogger('setPANID() Error: ' + str(e))
 
     def getPANID(self):
         """get current Thread Network PAN ID"""
-        print '%s call getPANID' % self.port
+        print('%s call getPANID' % self.port)
         return self.__sendCommand('panid')[0]
 
     def reset(self):
         """factory reset"""
-        print '%s call reset' % self.port
+        print('%s call reset' % self.port)
         try:
             self._sendline('factoryreset')
```

Comment:
Can we add a small delay after factory reset command? E.g. `time.sleep(0.5)`, without it the next command likes to fail.

Reply:
sure. will add.